apps/src/StudioApp.js

```diff
@@ -160,6 +160,20 @@ StudioApp.prototype.configure = function (options) {
   // currently mutually exclusive.
   this.editCode = options.level && options.level.editCode;
   this.usingBlockly_ = !this.editCode;
+
+  /** @member {boolean} Usually true; only false if this app breaks the
+   * conventional visualization+editor layout, and needs to skip much of
+   * StudioApp's layout control code (e.g. netsim)
+   */
+  this.usesStandardLayout_ = !options.usesCustomLayout;
+
+  // TODO (bbuchanan) : Replace this editorless-hack with setting an editor enum
+  // or (even better) inject an appropriate editor-adaptor.
```

Comment:
I'm fine closing this PR with this in it. I just want to avoid the case where we commit a TODO, then it drops off of our radar and we never address it (something I'm sometimes bad about).